library/init/algebra/field.lean

```diff
@@ -185,12 +193,26 @@ lemma one_div_div (a b : α) : 1 / (a / b) = b / a :=
 by rw [one_div_eq_inv, division_def, mul_inv',
        inv_inv', division_def]
 
+lemma div_helper {a : α} (b : α) (h : a ≠ 0) : (1 / (a * b)) * a = 1 / b :=
```

Comment:
```suggestion
lemma div_mul_mul_eq_div {a : α} (b : α) (h : a ≠ 0) : (1 / (a * b)) * a = 1 / b :=
```

But I'm not confident of my ability to choose good names.

Reply:
I assumed that this is a name used by `norm_num` or something, so I didn't touch it.

Reply:
Oh wow. That's the name we already use. I guess it's ok then.